routes/docs/category.js

```diff
@@ -1,26 +1,48 @@
 const processes = require('../../data/processes.json')
 const versions = require('../../data/versions.json')
 
-module.exports = (req, res) => {
+module.exports = (req, res, next) => {
+  const categoryList = [
+    { path: 'all', name: 'All the Electron Docs!' },
+    { path: 'tutorial', name: 'Guides' },
+    { path: 'api', name: 'API' },
+    { path: 'development', name: 'Development' },
+    { path: 'versions', name: 'Versions' }
+  ]
   const category = req.params.category
+  const selectedCategories = categoryList.filter(cat => cat.path === category)
   let context
-  if (['tutorial', 'api', 'development'].includes(category)) {
-    const categories = {
-      'tutorial': 'Guides',
-      'api': 'API',
-      'development': 'Development'
-    }
 
-    context = Object.assign(req.context, {
-      layout: 'docs',
-      processes: processes,
-      category: categories[category]
-    })
-  } else if (category === 'versions') {
-    context = Object.assign(req.context, {
-      layout: 'docs',
-      versions: versions
-    })
+  if (selectedCategories.length > 0) {
+    const selectedCategory =  selectedCategories[0]
+    if (category === 'all') {
+      context = Object.assign(req.context, {
+        page: {
+          title: `${selectedCategory.name} | Electron`
+        },
+        layout: 'docs',
+        viewingAllDocs: true
+      })
+    } else if (category === 'versions') {
+      context = Object.assign(req.context, {
+        page: {
+          title: `${selectedCategory.name} | Electron`
+        },
+        layout: 'docs',
+        versions: versions
+      })
+    } else {
+      context = Object.assign(req.context, {
+        page: {
+          title: `${selectedCategory.name} | Electron`
+        },
+        layout: 'docs',
+        processes: processes,
+        category: selectedCategory.name
+      })
+    }
+    res.render(`docs/${category}`, context)
+  }  else {
+    next()
```

Comment:
I would prefer an early return at the top, for readability:

```js
if (!selectedCategories.length) return next()
```

Also, can you add a test(s) for this? See test/index.js